client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -117,7 +117,7 @@ export class ImageEditorToolbar extends Component {
 		const items = [
 			{
 				action: AspectRatios.FREE,
-				label: translate( 'Free' ),
+				label: translate( 'Freeform', {context: 'Option in image editor used to crop images using freeform aspect ratio'} ),
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?